src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -0,0 +1,55 @@
+package mate.jdbc.model;
+
+public class Manufacturer {
+    private Long id;
+    private String name;
+    private String country;
+
+    public Manufacturer() {
+
+    }
+
+    public Manufacturer(Manufacturer manufacturer) {
+        this.id = manufacturer.getId();
+        this.name = manufacturer.getName();
+        this.country = manufacturer.getCountry();
+    }
```

Comment:
```suggestion
    public Manufacturer(Manufacturer manufacturer) {
    	this(manufacturer.getId(), manufacturer.getName(),manufacturer.getCountry() )
    }
```

It's better to use existing constructor